bin/propolis-server/src/lib/stats.rs

```diff
@@ -3,7 +3,10 @@
 // Copyright 2022 Oxide Computer Company
 
 use anyhow::anyhow;
-use dropshot::{ConfigDropshot, ConfigLogging, ConfigLoggingLevel};
+use dropshot::ConfigDropshot;
+use dropshot::ConfigLogging;
+use dropshot::ConfigLoggingLevel;
+use dropshot::HandlerTaskMode;
```

Comment:
Did rustfmt convert this from the parenthetical style?

Reply:
Do you mind keeping the old style, here and in `main.rs` too?

Reply:
Not at all, reverted in 88d7a53